src/components/user-form/AddUserForm.js

```diff
@@ -82,17 +99,26 @@ class AddUserForm extends Component {
             id="lastNameInput"
             placeholder="Example input"
           />
+          <input
+            ref={this.addressNameRef}
+            type="text"
+            className="form-control"
+            id="addressNameInput"
+            placeholder="Example Address"
+          ></input>
         </div>
 
+
         {/*
         todo 1: добавить инпут для ввода адресса
               передать ему атрибут под название ref наш созданный в строке 8 ref
         */}
 
         <button type='submit' className="btn btn-primary m-2">Add </button>
         <button type='button' onClick={this.focusInput}>focus</button>
+        <button type='reset' className="btn btn-danger" onClick={() => this.onReset} >Reset</button>
```

Comment:
try to do this:  remove  type='reset' and test what would happen with behavior
and get back to me and tell what the result you will get